CONTRIBUTING.md

```diff
@@ -29,6 +29,10 @@ To test the `rss` package, run `go test ./...`.
 
 Make your change, with new passing tests.
 
+See the change in a web browser by
+installing the package with `go install` and
+running the server with `foreman start`.
```

Comment:
Foreman is pretty foreign (hehe) to Go. I'd say just sticking with `go run main.go` is ideal here.